Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```diff
@@ -12,7 +12,7 @@
     netsync: false
     layers:
       - state: beaker
-      - state: beaker1  
+      - state: beaker1
```

Comment:
Why did you change the layers here?

Reply:
whoops

Reply:
i think it was rider editorconfig, don't see the harm in keeping the changes since its just whitespace removal